src/components/Modal/Modal.stories.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { useState } from 'react'
+
+import { Story } from '@storybook/react'
+
+import { Button } from 'components/Button'
+import { IconPlaylists } from 'components/Icons'
+
+import { Modal } from './Modal'
+import { ModalContent } from './ModalContent'
+import { ModalHeader, ModalTitle } from './ModalHeader'
+import { ModalProps } from './types'
+
+export default {
+  component: Modal,
+  title: 'Components/Modal'
+}
+
+const Template: Story<ModalProps> = ({
+  children = <>This is a modal!</>,
+  ...args
+}) => {
+  const [isOpen, setIsOpen] = useState(false)
+  return (
+    <>
+      <Button text='Click me' onClick={() => setIsOpen(true)} />
+      <Modal {...args} isOpen={isOpen} onClose={() => setIsOpen(false)}>
+        {children}
+      </Modal>
+    </>
+  )
+}
+
+export const Base = Template.bind({})
+export const LotsOfContent = Template.bind({})
+const LongContent = () => (
```

Comment:
do you think it is expected behavior from design system perspective to have a modal that doesn't have a header with some title? just thinking it might be good to have the first example the full case that we'd expect devs to adhere to 90% of the time?

Reply:
yeah I'll change that! good call